### PR TITLE
feat(cleantextbutton): Added clean text button at license text field

### PR DESCRIPTION
**Pull request by rohitpandey49**
Signed-off-by: rohitpandey49 <rohit.pandey4900@gmail.com>

## Description

Added 'Clean Text' button in license text field for #2108 

### Changes

Added 'Clean Text' button

## How to test

1. Go to change concluded license 
2. Click on license text field

closing #2108 

Please consider using the closing keyword if the pull request is proposed to
fix an issue already created in the repository
(https://help.github.com/articles/closing-issues-using-keywords/)

<a href="https://gitpod.io/#https://github.com/fossology/fossology/pull/2157"></a>

### CODE REVIEW

File: src/www/ui/scripts/change-license-common.js [hunk 1]
```diff
@@ -264,6 +264,13 @@ function openTextModel(uploadTreeId, licenseId, what, type) {
     $('#selectFromNoticeFile').css('display','none');
   }
 
+  if (what == 2 || what === 'reportinfo') {
+    // clicked to add button to display child modal
+    $('#clearText').css('display','inline-block');
+  } else {
+    $('#clearText').css('display','none');
```
Comment (GMishx) [inline src/www/ui/scripts/change-license-common.js:271]: Please consider using [$.show()](https://api.jquery.com/show/) and [$.hide()](https://api.jquery.com/hide/)

File: src/www/ui/scripts/change-license-common.js [hunk 2]
```diff
@@ -295,6 +302,12 @@ function closeTextModal() {
   textModal.modal('hide');
 }
 
+function cleanTextModal() {
+  var textAreaValue = document.getElementById("referenceText").value;
+  textAreaValue = textAreaValue.replace(/,/g, '');
+  document.getElementById("referenceText").value = textAreaValue;
+}
```
Comment (GMishx) [inline src/www/ui/scripts/change-license-common.js:309]: Please try to reuse following function https://github.com/fossology/fossology/blob/36db1bec85d1386d1050e87d9c3aeb1fcc43f103/src/www/ui/scripts/change-license-view.js#L97-L116

A simpler solution will be to pass value for `textField` as a parameter to `cleanText()`. What do you think?

**Review by GMishx — CHANGES_REQUESTED** [review]

File: src/www/ui/scripts/change-license-common.js (hunk 2 above)
Comment (rohitpandey49) [reply src/www/ui/scripts/change-license-common.js]: Sir, I have made the changes. Kindly check

File: src/www/ui/scripts/change-license-view.js [hunk 3]
```diff
@@ -94,8 +94,7 @@ function scheduleBulkScan() {
   scheduleBulkScanCommon($('#bulkIdResult'), reloadClearingTable);
 }
 
-function cleanText() {
-  var $textField = $('#bulkRefText');
+function cleanText($textField) {
```
Comment (GMishx) [inline src/www/ui/scripts/change-license-view.js:97]: I believe starting the variable with a `$` is a typo in JS. Can you please remove the `$` from `textField`?

Comment (GMishx) [reply src/www/ui/scripts/change-license-view.js:97]: And unfortunately, the function is duplicated in following place as well. Can you please update the same to have the same signature?

https://github.com/fossology/fossology/blob/4ce46a3ed1490dce672c3b02fea8aff930017f36/src/www/ui/scripts/change-license-browse.js#L136

**Review by GMishx — CHANGES_REQUESTED** [review]
Please consider following changes. And please squash your commits. We try to have atomic commit to reduce changelog length.

**Comment by rohitpandey49** [comment]
Closing this and starting a new pull request

**Pull request closed without merging**